src/Broadway/Bundle/BroadwayBundle/DependencyInjection/RegisterSagaCompilerPass.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Broadway\Bundle\BroadwayBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class RegisterSagaCompilerPass implements CompilerPassInterface
+{
+    private $multipleSagaManagerService;
+    private $tagName;
+
+    public function __construct($multipleSagaManagerService, $tagName)
+    {
+        $this->multipleSagaManagerService = $multipleSagaManagerService;
+        $this->tagName = $tagName;
+    }
+
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->has($this->multipleSagaManagerService)) {
+            throw new \LogicException(
+                sprintf('Unknown saga manager service known as %s', $this->multipleSagaManagerService)
+            );
+        }
+
+        $sagas = [];
```

Comment:
Broadway is PHP 5.3 :) I'm would not have a problem with a minimum PHP version bump.